assets/js/components/adminbar/AdminBarWidgets.stories.js

```diff
@@ -175,9 +208,14 @@ AnalyticsInactiveNew.decorators = [
 		);
 	},
 ];
-AnalyticsInactiveNew.parameters = {
+AnalyticsInactiveNewCompleteActivation.parameters = {
 	features: [ 'zeroDataStates' ],
 };
+AnalyticsInactiveNewCompleteActivation.scenario = {
+	label:
+		'Views/AdminBarApp/AdminBarWidgets/AnalyticsInactiveNewCompleteActivation',
+	delay: 3000,
```

Comment:
Same as above, but also I would hope we wouldn't need to add a delay for such simple components.